debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/ZZZGtidSetIT.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.fest.assertions.Assertions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.mysql.MySqlConnectorConfig.SnapshotMode;
+import io.debezium.doc.FixFor;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.Testing;
+
+/**
+ * The test is named to make sure it runs alphabetically last as it can influence execution
+ * of for example {@link BinlogReaderIT}
+ *
+ * @author Jiri Pechanec
+ */
+public class ZZZGtidSetIT extends AbstractConnectorTest {
```

Comment:
Uff, ok, that's kinda... ugly :)

Reply:
No that I am superrhappy about that, but it works and I inteded to visually make it declared it is in the end. It was working when the code was in MySsqlConnectorIT as the alphabetical order also gurateed it but I did not like to hide dependencies/.